src/index.html

```diff
@@ -10,5 +10,10 @@
 </head>
 <body>
   <h1>Stop watch</h1>
+  <div class="watch">
+    <div class="watch__center"></div>
+    
+    
+  </div>
```

Comment:
```suggestion
  <div class="watch">
    <div class="watch__center"></div>
  </div>
```
Don't leave empty lines